doc/release-notes.md

```diff
@@ -115,34 +167,47 @@ RPC Changes
   transparent addresses when available.
 - The result value for `z_importviewingkey` now includes an `address_type` field
   that replaces the now-deprecated `type` key.
+- `z_listunspent` has been updated to render unified addresses for Sapling and 
+  Orchard outputs when those outputs are controlled by unified spending keys.
+  Outputs received by unified internal addresses do not include the `address` field.
+- Legacy transparent address generation using `getnewaddress` no longer uses a 
+  preallocated keypool, but instead performs HD derivation from the wallet's mnemonic
+  seed according to BIP 39 and BIP 44 under account ID `0x7FFFFFFF`.
 
 ### 'z_sendmany'
 
-- The `z_sendmany` RPC call no longer permits Sprout recipients in the
-  list of recipient addresses. Transactions spending Sprout funds will
-  still result in change being sent back into the Sprout pool, but no
-  other `Sprout->Sprout` transactions will be constructed by the Zcashd
-  wallet.
-
-- The restriction that prohibited `Sprout->Sapling` transactions has been
-  lifted; however, since such transactions reveal the amount crossing
-  pool boundaries, they must be explicitly enabled via a parameter to
-  the `z_sendmany` call.
-
-- A new string parameter, `privacyPolicy`, has been added to the list of
-  arguments accepted by `z_sendmany`. This parameter enables the caller to
-  control what kind of information they permit `zcashd` to reveal when creating
-  the transaction. If the transaction can only be created by revealing more
-  information than the given strategy permits, `z_sendmany` will return an
-  error. The parameter defaults to `LegacyCompat`, which applies the most
-  restrictive strategy `FullPrivacy` when a Unified Address is present as the
-  sender or a recipient, and otherwise preserves existing behaviour (which
-  corresponds to the `AllowFullyTransparent` policy).
-
+- The `z_sendmany` RPC call no longer permits Sprout recipients in the list of recipient
```

Comment:
Then you have the same problem with `addresses`. Also, in general we're trying to shift our nomenclature to refer to "receivers" and "recipients" instead of "addresses" which, with UAs, are a less precise concept.